ansible/roles/test/tasks/everflow_testbed/apply_config/session.json

```diff
@@ -5,7 +5,7 @@
             "dst_ip": "2.2.2.2",
             "gre_type": "0x6558",
             "dscp": "50",
-            "ttl": "64",
+            "ttl": "1",
```

Comment:
can you add a comment on this, why change to ttl=1?

Reply:
it's a bit tricky to add comments in the JSON file. I've added enough information in the commit message. `git blame` is able to find out why this change is here.